react/AddressForm.tsx

```diff
@@ -108,30 +108,42 @@ const AddressForm: StorefrontFunctionComponent = () => {
 
   const parseLineFragment = (fragment: LineFragment) => {
     const field = fields[fragment.name as keyof Fields]
+
     if (
       !field ||
       ignoredFields.has(fragment.name) ||
       address[fragment.name] == null
     )
       return null
 
+    const style = styleRules[field.label]
+
     if (hasWithoutNumberOption(field.label))
-      return <NumberOption showCheckbox />
+      return (
+        <div className="flex-auto mh3" style={style as React.CSSProperties}>
+          <NumberOption showCheckbox />
+        </div>
+      )
```

Comment:
Missing curly braces is kinda weird with multiple lines 🤔

Reply:
Always use curly braces in `if`  statements, even if the body has a single statement, as that avoids bugs in case someone adds something to this `if` block in the future.

Reply:
I used to always add curly braces to `if` statements for that exact reason 😅 I'll fix it.